bungee/src/main/java/com/sekwah/advancedportals/bungee/connector/container/BungeeProxyContainer.java

```diff
@@ -27,9 +28,10 @@ public void invokeCommand(ProxyPlayerContainer proxyPlayer,
     public void transferPlayer(ProxyPlayerContainer proxyPlayer,
                                String serverName) {
         // Should never not be true but just to be safe
-        if (proxyPlayer instanceof BungeeProxyPlayerContainer playerContainer) {
-            var serverInfo = plugin.getProxy().getServerInfo(serverName);
-            var player = playerContainer.getPlayer();
+        if (proxyPlayer instanceof BungeeProxyPlayerContainer) {
+            BungeeProxyPlayerContainer playerContainer = (BungeeProxyPlayerContainer) proxyPlayer;
+            net.md_5.bungee.api.config.ServerInfo serverInfo = plugin.getProxy().getServerInfo(serverName);
```

Comment:
For all of these dont use direct imports inline, make sure the imports are at the top of the file unless they clash with another already imported name.